modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/SelectWithOption.tsx

```diff
@@ -12,23 +12,52 @@
  * details.
  */
 
-import {ClaySelect, ClaySelectWithOption} from '@clayui/form';
-import React from 'react';
+import ClayAutocomplete from '@clayui/autocomplete';
+import DropDown from '@clayui/drop-down';
+import ClayIcon from '@clayui/icon';
+import classNames from 'classnames';
+import React, {useRef, useState} from 'react';
 
 import {FieldBase} from './FieldBase';
 
+import './Select/index.scss';
+
+interface SelectWithOptionProps
+	extends React.SelectHTMLAttributes<HTMLSelectElement> {
+	ariaLabel?: string;
+	disabled?: boolean;
+	error?: string;
+	feedbackMessage?: string;
+	items: Item[];
+	label?: string;
+	onSelectChange: (label: string, value: string) => void;
+	required?: boolean;
+	tooltip?: string;
+}
+
+type Item = {
```

Comment:
@Manda-vit Please, declare this type before the interface `SelectWithOptionProps`, since the interface is using it. It's a good practice to keep the declarations before the usage :)